### ISSUE TITLE
Resolve error: require(...).promisify is not a function (import/no-unresolved)

### ISSUE THREAD

**Issue by sebj54** [vuejs/vue-cli#1454]
### Version
3.0.0-beta.15

### Reproduction link
[https://codesandbox.io/s/vue](https://codesandbox.io/s/vue)

### Steps to reproduce
Create a sample Vue project.
Open project in VS Code with eslint plugin.
Open "Home.vue" file, wait for errors to come.

### What is expected?
No error in "Problems" tab.

### What is actually happening?
Several eslint import problems like these:

```
Resolve error: require(...).promisify is not a function (import/no-unresolved)
Resolve error: require(...).promisify is not a function (import/no-extraneous-dependencies)
Resolve error: require(...).promisify is not a function (import/no-duplicates)
Resolve error: require(...).promisify is not a function (import/extensions)
Resolve error: require(...).promisify is not a function (import/no-named-as-default)
Resolve error: require(...).promisify is not a function (import/no-named-as-default-member)
Unable to resolve path to module '@/components/HelloWorld.vue'. (import/no-unresolved)
```

---
Please tell me if you need more information (especially versions but nearly everything is up to date), I'll be happy to help!

**Comment by mehwww** [comment]
It seems cuz node ship with electorn which vscode using version is 7.9.0, and [this line](https://github.com/vuejs/vue-cli/blob/cd88b47edcc102903e787d55b238a7f5e9e1c900/packages/%40vue/cli-shared-utils/index.js#L17) use the new promisify api ship with node 8.x and break the eslint, use a vendor lib do the promisify work can resolve this issue.

**Comment by sebj54** [comment]
Thanks!
Your workaround is working for me, but it seems like unresolved paths are not reported by eslint anymore. It is not so important because webpack throws errors if paths are wrong.

**Comment by yyx990803** [comment]
FYI, I can't reproduce the problem with a project created by the latest CLI + VSCode + ESLint plugin.

It's weird because your editor's ESLint plugin should not attempt to lint library code, so that's definitely a configuration problem for your plugin.

**Comment by thomasmichaelwallace** [comment]
I think you were premature closing this issue. I've hit it, and can trace it directly to the line/cause @mehwww states:
https://github.com/vuejs/vue-cli/issues/1473

**Comment by sebj54** [comment]
> ESLint plugin should not attempt to lint library code

It's actually linting the sample project's code. I don't understand your point of view. It's clear here that this project and the plugin are incompatible. I'm just trying to get a fix (it can also be a extra configuration of the plugin) **and** get that fix documented if it needs to be. 

Thanks @thomasmichaelwallace  for your extra information! I hope it will help to find a solution.

**Comment by chanlito** [comment]
Suddenly my project is showing this error too. Dunno how to fix it atm.

**Comment by thomasmichaelwallace** [comment]
Just so you know, this got fixed under #1473 .
